news/2024/2024-06-17-the-followpoint-dressurf-top-keyboard-smasher

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "The Followpoint: dressurf, Top Keyboard Smasher"
+date: 2024-06-17 17:00:00 +0000
+series: The Followpoint
+---
+
+It's the latest in a series of top player interviews, now featuring the osu!mania #1 best keyboard smasher: [dressurf](https://osu.ppy.sh/users/758406)!
+
+![](/wiki/shared/news/2024-06-17-the-followpoint-dressurf-top-keyboard-smasher/banner.jpg)
+
+That's right, following previous interviews with [mrekk](https://osu.ppy.sh/home/news/2024-05-06-the-followpoint-mrekk-the-osu-standard-prodigy), [YesMyDarknesss](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch) and [dressurf](https://osu.ppy.sh/users/758406), top players of osu!, osu!catch, and osu!taiko respectively, this edition of The Followpoint continues the pattern by featuring osu!mania's rank #1 player and the [osu!mania 7K World Cup 2024](/wiki/Tournaments/MWC/2024_7K) champion.
+
+Simply read on for a look into [dressurf](https://osu.ppy.sh/users/758406)'s insight:
+
+---
+
+*For the people who don’t know who you are, how would you introduce yourself?*
+
+> Thanks for having me for this event, The Followpoint, my name is dressurf, and I'm constantly striving to to improve my skills furthermore.
+
+*How did you get introduced to osu!, and what was your first impression of it?*
+
+> It all began when I played Osu! Tatakae! Ouendan (Elite Beat Agents) on the Nintendo DS Lite. I was immediately intrigued by its unique gameplay style. It didn't take too long to find out that there's a similar rhythm game named "osu!".

```

Comment:
```suggestion
> It all began when i played Osu! Tatakae! Ouendan (Elite Beat Agents) on the Nintendo DS Lite. I was immediately intrigued by its unique gameplay style. It didn't take too long to find out that there's a similar rhythm game named "osu!".

```

Reply:
"I" is supposed to be capitalised in english